src/material-experimental/mdc-slider/slider.html

```diff
@@ -0,0 +1,25 @@
+<!-- Inputs -->
+<ng-content></ng-content>
+
+<!-- Track -->
+<div class="mdc-slider__track">
+  <div class="mdc-slider__track--inactive"></div>
+  <div class="mdc-slider__track--active">
+    <div class="mdc-slider__track--active_fill" #trackActive></div>
+  </div>
+  <div *ngIf="showTickMarks" class="mdc-slider__tick-marks" #tickMarkContainer>
+    <div *ngFor="let tickMark of tickMarks" class="{{_getTickMarkClass(tickMark)}}"></div>
```

Comment:
```suggestion
    <div *ngFor="let tickMark of tickMarks" [class]="_getTickMarkClass(tickMark)"></div>
```
Prefer property bindings over text interpolation